demo/src/routes/[framework]/components/accordion/playground/+page.svelte

```diff
@@ -3,6 +3,33 @@
 	import {getAccordionDefaultConfig} from '@agnos-ui/core';
 	import sample from '@agnos-ui/samples/accordion/playground';
 	import doc from '../../../../../../../core/dist/api.json?accordion&extractApi';
+	import type {SelectValues} from '@agnos-ui/common/propsValues';
+	export let listPropsValues: {[key in keyof ReturnType<typeof getAccordionDefaultConfig>]?: SelectValues[]} = {
+		onShown: [
+			{value: 'noop', label: 'Noop'},
```

Comment:
Instead of repeating everywhere the value and label, wouldn't it be better to accept here only the value, and keep the label in a central place (along with the function definition)?

Reply:
Indeed makes sense since there are lot of reused labels